e2e/tests/regression/donation-flow/anon-donation-custom.spec.ts

```diff
@@ -0,0 +1,138 @@
+import { test, expect, Page } from '@playwright/test'
+import { HeaderPage } from '../../../pages/web-pages/header.page'
+import { HomePage } from '../../../pages/web-pages/home.page'
+import { CampaignsPage } from '../../../pages/web-pages/campaigns/campaigns.page'
+import { bgLocalizationOneTimeDonation } from '../../../data/localization'
+import { DonationPage } from '../../../pages/web-pages/campaigns/donation.page'
+import { bgDonationRegions } from '../../../data/enums/donation-regions.enum'
+import { StripeCheckoutPage } from '../../../pages/web-pages/external/stripe-checkout.page'
+import { anonDonationTestData } from '../../../data/support-page-tests.data'
+import { campaignData } from '../../../data/campaigns-env.data'
+
+// This spec contains E2E tests related to anonymous donation flow - custom amount
+// The tests are dependent, the whole describe should be runned
+test.describe.serial(
+  'Anonymous contributor is able to donate custom amount - BG language version',
+  async () => {
+    let page: Page
+    let homepage: HomePage
+    let headerPage: HeaderPage
+    let campaignsPage: CampaignsPage
+    let donationPage: DonationPage
+    let stripeCheckoutPage: StripeCheckoutPage
+    // For the URL is used RegExp so the tests will pass for each environment - localhost, dev, etc.
+    const firstCampaignPageUrl = `/campaigns/${campaignData[0].slug}`
```

Comment:
[change requested]
the first campaign should come by selecting from the list of existing campaigns, because the predefined hard-coded title or slug which may not exist in all environments

Reply:
Check https://github.com/podkrepi-bg/frontend/pull/1265/commits/ee7116a07564a6b0f11c134add3a765966acc3fa